mods/ts/sequences/structures.yaml

```diff
@@ -862,6 +862,10 @@ proc: # TODO: unused narefn_a, narefn_b
 	make: ntrefnmk
 		Length: 20
 		ShadowStart: 20
+	dock: ntrefn_a # TODO: unused
```

Comment:
Well *some* version of Tiberian Sun is using it - https://youtu.be/iD_CSM_6Alg?t=203